pages/linux/eval.md

```diff
@@ -0,0 +1,11 @@
+# eval
+
+> Execute arguments as a single command in the current shell and return its result.
+
+- Call `echo` with the `foo` argument:
```

Comment:
Put foo inside double quotes and remove backticks. It is mainly used to denote commands.